release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,44 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+- Due to Azure limitations, upgraded clusters do not support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
+
+---
+
+### azure-operator v2.9.0
+
+### Kubernetes v1.16.3
+
+### Calico v3.10.1
+
+### cert-exporter [v1.2.1](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#121-2019-12-24)
+
+### chart-operator [v0.11.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.11.4)
+- Now handling timeouts by pulling chart tarballs.
+
+### cluster-operator [v0.21.4](https://github.com/giantswarm/cluster-operator/releases/tag/v0.21.4)
+
+### coreDNS v1.6.5 ([Giant Swarm app v1.1.3](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v113))
+- Updated HPA manifests for Kubernetes 1.16.
+
+### CoreOS Container Linux v2247.6.0
+
+### etcd v3.3.17
+
+### external-dns v0.5.11 ([Giant Swarm app v1.1.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#v110))
+
+### Helm v2.16.1 (primarily for Giant Swarm internal use)
+
+### kube-state-metrics v1.9.2 ([Giant Swarm app v1.0.2](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v102))
+
+### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
+- Allow disabling DNS TCP check.
+
+### nginx-ingress-controller v0.28.0 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v130-2020-01-30))
```

Comment:
Is this right? I couldn't find it in any of the dependencies

Reply:
See comment above https://github.com/giantswarm/releases/pull/84/files#r374333444